.travis.yml

```diff
@@ -31,6 +36,9 @@ matrix:
       env: lint
       install: travis_retry pip install flake8==3.3.0
       script: flake8 --show-source
+  exclude:
+    - python: 2.7
```

Comment:
We need to exclude PyPy too

Reply:
I'm surprised, I didn't know they stopped supporting PyPy...

Reply:
PyPy in this context means the Python 2.7 version. I suppose what really ought to do is to add a PyPy3 environment as well.

Reply:
Ah yes of course! Yes, but _I think_ Django doesn't support for PyPy3 officially (that might no longer be true though...)